python/lsst/obs/base/gen2to3/translators.py

```diff
@@ -19,7 +19,9 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-__all__ = ("Translator", "KeyHandler", "CopyKeyHandler", "ConstantKeyHandler",
+from __future__ import annotations
+
+__all__ = ("Translator", "TranslatorFactory", "KeyHandler", "CopyKeyHandler", "ConstantKeyHandler",
```

Comment:
Do we actually need to export `Translator` here? It seems like all the specialization happens when an Instrument configures its Factor.

Reply:
Hmm.  Would be best to export it here, I think, but not in the package-level `__init__.py`.  Will do that.